src/business/.gitignore

```diff
@@ -0,0 +1,15 @@
+/Address.class
```

Comment:
remove this file entirely... the `*.class` line inside the `.gitignore` file on the root path should be sufficient